app/docs/md/learn/concepts/styling/index.md

```diff
@@ -2,64 +2,45 @@
-
-<doc-callout level="none" mark="🖍️">
-
-**[Read more about Enhance utility classes →](/docs/learn/concepts/styling/utility-classes)**
-
-</doc-callout>
-
-## Logical properties
-
-Enhance Styles’ utility class system is built with logical properties in mind. These properties make it easy to compose styles that are resilient across multiple writing modes.
-
-<doc-callout level="none" mark="🔀">
-
-**[Read more about logical properties →](/docs/learn/concepts/styling/logical-properties)**
+**[Read more about using plain old CSS →](/docs/learn/concepts/styling/css)**
 
 </doc-callout>
 
-## Customization
+## Tailwind
 
-Enhance Styles is built with parametric design systems in mind. A huge number of configurable options are available, with sensible defaults baked into all new projects.
+You can use whatever CSS tooling you are used to. Just configure it to output a CSS file to the public folder then reference it in your [`head`](/docs/learn/starter-project/head).
 
-<doc-callout level="none" mark="🎛️">
+<doc-callout level="none" mark="🐈💨">
 
-**[Read more about customization →](/docs/learn/concepts/styling/customization)**
+**[Read more about using Tailwind →](/docs/learn/concepts/styling/using-tailwind)**
 
 </doc-callout>
 
-## Modular scales
+## Sass
 
-For styling typography and layouts, Enhance Styles’ fluid, modular scales reduce the amount of breakpoint specific code you need to write, while allowing these styles to scale gracefully across a wide range of viewports.
+Again, use whatever CSS tooling you like. Just configure it to output a CSS file to the public folder then reference it in your [`head`](/docs/learn/starter-project/head).
 
-<doc-callout level="none" mark="📏">
+<doc-callout level="none" mark="📠 > 📠">
 
-**[Read more about modular scales→](/docs/learn/concepts/styling/modular-scales)**
+**[Read more about using Sass →](/docs/learn/concepts/styling/using-sass)**
 
 </doc-callout>
 
-## Element styles
+## Enhance styles
 
-Certain styles would not be appropriate as a utility class. For these cases, you can write specific styles using the `<style>` tag in your custom element. These styles will be scoped to the element they’re declared in (and its children), and can take advantage of Enhance Styles’ generated custom properties.
+Enhance is a "batteries included" full-stack framework so we've preconfigured a styling system that is the culmination of decades of CSS research and development into a "best of all worlds" approach to styling. It isn't what all the cool kids are using... yet, but we feel that if you give it a chance the benefits of our **deep** styling experience will become apparent.
```

Comment:
This sentence is a bit long and complex; suggested edit to make it more concise without losing the information:

```suggestion
Enhance is a "batteries included" full-stack framework and comes with a preconfigured (and customizable) "best of all worlds" styling system, comprising decades of CSS research and development. It isn't what all the cool kids are using — yet! — but we feel that if you give it a chance, the benefits of our **deep** styling experience will become apparent.
```